ConsumerEdition/HiKey/Installation/LinuxFastboot.md

```diff
@@ -17,7 +17,7 @@ This section shows how to install new operating system to your HiKey using the f
 - The Linux “Tools Only” SDK download does not come with fastboot, you will need to use the Android SDK Manager to install platform-tools.
 - To do this follow the “SDK Readme.txt” instructions included in your SDK “Tools Only” download.
 
-If you are still having trouble setting up fastboot, <a href="https://youtu.be/W_zlydVBftA" target="_blank">click here</a> for a short tutorial video
+Setup Fastboot using this command: sudo apt-get install android-tools-fastboot
```

Comment:
I have a new way to setup fastboot tools that stumps all old ways. I will make this update.